TranslatorReasonerAPI.yaml

```diff
@@ -470,46 +470,75 @@ components:
+            expressed in the edge.
+          example: Biolink:has_p-value_evidence, Biolink:has_original_source
+        original_attribute_name:
           type: string
           description: >-
-            Human-readable name or label for the attribute. If appropriate,
-            should be the name of the semantic type term.
-          example: PubMed Identifier
+            The term used by the original source of an attribute to describe
+            the meaning or significance of the value it captures. This may be
+            a column name in a source tsv file, or a key in a source json
+            document for the field in the data that held the attribute's
+            value. Capturing this information  where possible lets us preserve
+            what the original source said. Note that the data type is string'
+            but the contents of the field could also be a CURIE of a third
+            party ontology term.
+          example: p-value
           nullable: true
         value:
-          example: 32529952
           description: >-
             Value of the attribute. May be any data type, including a list.
-        type:
+          example: 0.000153
+        value_type_id:
           $ref: '#/components/schemas/CURIE'
           description: >-
-            CURIE of the semantic type of the attribute. For properties
-            defined by the Biolink model this should be a biolink CURIE,
-            otherwise, if possible, from the EDAM ontology. If a suitable
-            identifier does not exist, enter a descriptive phrase here and
-            submit the new type for consideration by the appropriate
-            authority.
+            CURIE describing the semantic type of an  attribute's value. Use
+            a Biolink class if possible, otherwise a term from an external
+            ontology. If a suitable CURIE/identifier does not exist, enter a
+            descriptive phrase here and submit the new type for consideration
+            by the appropriate authority.
           example: EDAM:data_1187
-        url:
+        attribute_source:
+          type: string
+          description: >-
+            The source of the core assertion madeby the key-value pair of an
+            attribute object. Use a CURIE or namespace designator for this
+            resource where possible.
+          example: UniProtKB
+          nullable: true
+        value_url:
           type: string
           description: >-
-            Human-consumable URL to link out and provide additional information
-            about the attribute (not the node or the edge).
+            Human-consumable URL linking to a web document that provides
+            additional information about an  attribute's value (not the node
+            or the edge fom which it hangs).
           example: https://pubmed.ncbi.nlm.nih.gov/32529952
           nullable: true
-        source:
+        description:
```

Comment:
Is there a conflict with shadowing names from outer scope? Consider renaming to value_url_description if so.

Reply:
The way that github shows changes here is a bit confusing.  The 'description' field on line 534 in green here is meant to hold a description of the info captured by the attribute. Nothing to do with the value_url.   The fork in Richard B's repo that this PR is based on may provide a clearer view of the current proposal (absent of github's interpretation of line changes): https://github.com/STARInformatics/ReasonerAPI/blob/issue-185-attribute-schema-revisions_R1.1/TranslatorReasonerAPI.yaml#L470